includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2125,4 +2125,22 @@ public function get_upe_available_payment_methods() {
 			]
 		);
 	}
+
+	/**
+	 * Text provided to users during onboarding setup.
+	 *
+	 * @return string
+	 */
+	public function get_setup_help_text() {
+		return __( 'Next we’ll ask you to share a few details about your business to create your account.', 'woocommerce-payments' );
+	}
+
+	/**
+	 * Get the oAuth connection URL.
+	 *
+	 * @return string Connection URL.
+	 */
+	public function get_oauth_connection_url() {
```

Comment:
Just a heads up that this may be updated to `get_connection_url()` in https://github.com/woocommerce/woocommerce-admin/pull/7095 and `$return_url` will be passed here for use.

Reply:
Updated 👍🏻